scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2558,127 +2678,13 @@ s32 BPF_STRUCT_OPS_SLEEPABLE(layered_init)
 	dbg("CFG: Dumping configuration, nr_online_cpus=%d smt_enabled=%d little_cores=%d",
 	    nr_online_cpus, smt_enabled, has_little_cores);
 
+	u64 fallback_dsq_id = 0;
```

Comment:
nit: shouldn't that be some FALLBACK_DSQ_ID defined somewhere?

Reply:
So this is just the counter that is used to sequentially allocate the hi fallback DSQs.